### ISSUE TITLE
Duplicates in the Master EverDrive SMDB

### ISSUE THREAD

**Issue by frederic-mahe** [SmokeMonsterPacks/EverDrive-Packs-Lists-Database#54]
```
3bdc8ef0dd64f58ce76815fae7669d7b3f371c18c4e24ea87356f78c55cf95bb	Master EverDrive/4 Homebrew/Gekioko PunPun Maru (PD) Future Driver.sms
3bdc8ef0dd64f58ce76815fae7669d7b3f371c18c4e24ea87356f78c55cf95bb	Master EverDrive/4 Homebrew/Gekioko PunPun Maru 2013-12-15.sms

421b3e613dec75310d925ace72434917a622c7083781b1fe396e94e296f94080	Master EverDrive/3 Sega SG-1000 Computer Videogame/Choplifter (Japan) (Beta).sg
421b3e613dec75310d925ace72434917a622c7083781b1fe396e94e296f94080	Master EverDrive/3 Sega SG-1000 Computer Videogame/Choplifter (SG-1000) (Prototype).sg

5bc9da3071f2af6bbeaa9030ace94402ee618fe67c25893048ddd992a22c53f9	Master EverDrive/4 Game Series Collections/Controllers & Peripherals/Genesis Controller Patched/Wonder Boy in Monster World Color Restore+Save+MD Controller Detect (Hack) v1 ichigobankai.sms
5bc9da3071f2af6bbeaa9030ace94402ee618fe67c25893048ddd992a22c53f9	Master EverDrive/4 Game Series Collections/Controllers & Peripherals/Genesis Controller Patched/Wonder Boy in Monster World Color Restore+Save+SMS-MD Controller Detect (Hack) v1 ichigobankai.sms

5bc9da3071f2af6bbeaa9030ace94402ee618fe67c25893048ddd992a22c53f9	Master EverDrive/4 Game Series Collections/Save Patched (SRAM)/Wonder Boy in Monster World Color Restore+Save+MD Controller Detect (Hack) v1 ichigobankai.sms
5bc9da3071f2af6bbeaa9030ace94402ee618fe67c25893048ddd992a22c53f9	Master EverDrive/4 Game Series Collections/Save Patched (SRAM)/Wonder Boy in Monster World Color Restore+Save+SMS-MD Controller Detect (Hack) v1 ichigobankai.sms

5bc9da3071f2af6bbeaa9030ace94402ee618fe67c25893048ddd992a22c53f9	Master EverDrive/4 Hacks/Controller & Light Gun Patches/Genesis Controller Patched/Wonder Boy in Monster World Color Restore+Save+MD Controller Detect (Hack) v1 ichigobankai.sms
5bc9da3071f2af6bbeaa9030ace94402ee618fe67c25893048ddd992a22c53f9	Master EverDrive/4 Hacks/Controller & Light Gun Patches/Genesis Controller Patched/Wonder Boy in Monster World Color Restore+Save+SMS-MD Controller Detect (Hack) v1 ichigobankai.sms

68065f54e21d0a0c90d407e1dc539c549cb325ad6fba86a13788debf141edec4	Master EverDrive/2 Other Regions A-Z/Korea - A-Z/Super Columns & Tetris (Korea) (Unl).sms
68065f54e21d0a0c90d407e1dc539c549cb325ad6fba86a13788debf141edec4	Master EverDrive/2 Other Regions A-Z/Korea - A-Z/Super Columns (KR).sms

85a47f7bfbdafb132280f8db6ba1eb78a56b5d0b44c2df997dabafc23fe088ad	Master EverDrive/4 Hacks/Bubble Bobble Improvement (Europe) v1 Penta Penguin.sms
85a47f7bfbdafb132280f8db6ba1eb78a56b5d0b44c2df997dabafc23fe088ad	Master EverDrive/4 Hacks/Bubble Bobble Improvement (Japan) v1 Penta Penguin.sms

8c9d6b995b241b50bb555418fe4fcb99efb1bcc19ee9148ed5b795702e2ac280	Master EverDrive/3 Game Gear Conversions/Dream Eleven v1 Revo.sms
8c9d6b995b241b50bb555418fe4fcb99efb1bcc19ee9148ed5b795702e2ac280	Master EverDrive/3 Game Gear Conversions/J.League Soccer Dream Eleven v1 Revo.sms

932a7d945822091219532b78cd34d976bb67d74c0c08fd844e1b88cf831af32b	Master EverDrive/4 Game Series Collections/Controllers & Peripherals/FM Sound Support/All FM Sound Supported Games/Golfamania (Europe).sms
932a7d945822091219532b78cd34d976bb67d74c0c08fd844e1b88cf831af32b	Master EverDrive/4 Game Series Collections/Controllers & Peripherals/FM Sound Support/All FM Sound Supported Games/Golfamania (Europe).sms copy

b808228b7775b427d52c90ac098cd750baf80a9da6541f6cb5fe1b85a64a7157	Master EverDrive/4 Betas, Prototypes, Revisions/Prototypes/Hook (Europe) (Proto).sms
b808228b7775b427d52c90ac098cd750baf80a9da6541f6cb5fe1b85a64a7157	Master EverDrive/4 Betas, Prototypes, Revisions/Prototypes/Hook (Prototype) [!].sms

b86acd2596750e5f51cb5edf72072319230ed22bba371aeb211f74b4d18aba5d	Master EverDrive/2 Other Regions A-Z/Brazil - A-Z/As Aventuras da TV Colosso (B).sms
b86acd2596750e5f51cb5edf72072319230ed22bba371aeb211f74b4d18aba5d	Master EverDrive/2 Other Regions A-Z/Brazil - A-Z/TV Colosso (Brazil).sms

ce73fc27f2e26507154c0a41e53a4c17e000f8e713e856f3109fbaa7e8dd32cb	Master EverDrive/4 Game Series Collections/Genre/Shoot 'em Up/All Shoot 'em Ups/Blade Eagle (World).sms
ce73fc27f2e26507154c0a41e53a4c17e000f8e713e856f3109fbaa7e8dd32cb	Master EverDrive/4 Game Series Collections/Genre/Shoot 'em Up/All Shoot 'em Ups/Blade Eagle 3D.sms

ce73fc27f2e26507154c0a41e53a4c17e000f8e713e856f3109fbaa7e8dd32cb	Master EverDrive/4 Game Series Collections/Genre/Shoot 'em Up/Sort By - View - Vertical/Blade Eagle (World).sms
ce73fc27f2e26507154c0a41e53a4c17e000f8e713e856f3109fbaa7e8dd32cb	Master EverDrive/4 Game Series Collections/Genre/Shoot 'em Up/Sort By - View - Vertical/Blade Eagle 3D.sms

ec0bfe582d35a31900d41d85905bc2c531e5420e76eada16379078938bc05144	Master EverDrive/2 Other Regions A-Z/Brazil - A-Z/California Games II (Brazil, Korea).sms
ec0bfe582d35a31900d41d85905bc2c531e5420e76eada16379078938bc05144	Master EverDrive/2 Other Regions A-Z/Brazil - A-Z/Jogos de Verao II (B) [!].sms
```

**Comment by frederic-mahe** [comment]
@SmokeMonsterPacks was this corrected in the latest pack release?

**Comment by SmokeMonsterPacks** [comment]
Doh! I will get the dupes fixed in the next updates. I was in such a hurry that I forgot to address them.

**Comment by frederic-mahe** [comment]
fixed in 5aaf1d0f3fb75a64051042e92cb168410b4bbda4
